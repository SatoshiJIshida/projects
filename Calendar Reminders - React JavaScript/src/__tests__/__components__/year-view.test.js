import React from "react";
import { shallow } from "enzyme";
import YearView from "../../components/year-view";

describe("Year View Component", () => {
  it("should be defined", () => {
    expect(YearView).toBeDefined();
  });

  it("should render correctly", () => {
    const newYearView = shallow(<YearView name="yearView-test" />);
    expect(newYearView).toMatchSnapshot();
  });
});
