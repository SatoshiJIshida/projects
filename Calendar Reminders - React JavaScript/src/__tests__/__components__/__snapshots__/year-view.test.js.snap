// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Year View Component should render correctly 1`] = `ShallowWrapper {}`;
